src/Command/Pull/PullDatabaseCommand.php

```diff
@@ -19,12 +19,14 @@ class PullDatabaseCommand extends PullCommandBase {
    * {inheritdoc}.
    */
   protected function configure() {
-    $this->setDescription('Import latest database backup from a Cloud Platform environment')
-      ->setHelp('This uses the latest available database backup, which may be up to 24 hours old. You can generate an on-demand backup using api:environments:database-backup-create.')
+    $this->setDescription('Import database backup from a Cloud Platform environment')
+      ->setHelp('This uses the latest available database backup, which may be up to 24 hours old. If no backup exists, one will be created.')
       ->setAliases(['pull:db'])
       ->acceptEnvironmentId()
       ->addOption('no-scripts', NULL, InputOption::VALUE_NONE,
         'Do not run any additional scripts after the database is pulled. E.g., drush cache-rebuild, drush sql-sanitize, etc.')
+      ->addOption('on-demand', NULL, InputOption::VALUE_NONE,
```

Comment:
Seems like this should be called —create or —create-backup. The —on-demand wording isn’t very self evident.

Reply:
Or maybe —new or —new-backup

Reply:
I don't think `--create` or `--new` would work. If we have to choose, `--create-backup` is the most obvious and IMHO works better than `--new-backup`.

"On-demand" is known terminology at Acquia to talk about backups, though, e.g. https://docs.acquia.com/cloud-platform/arch/security/availability/backups/#subscriber-on-demand-backups

Reply:
Ok I didn't realize this was known terminology.